packages/clerk-js/src/ui/contexts/ClerkUIComponentsContext.tsx

```diff
@@ -57,54 +61,41 @@ export const useSignUpContext = (): SignUpContextType => {
   const options = useOptions();
   const clerk = useClerk();
 
+  assertNoLegacyProp(options);
+  assertNoLegacyProp(ctx);
```

Comment:
We decided that we're not going to support only the new props going forwards - we're throwing if we detect any of the old props